firmware/integration/rusefi_config.txt

```diff
@@ -638,13 +643,13 @@ end_struct
 struct dc_io
 	brain_pin_e directionPin1;
 	brain_pin_e directionPin2;
-	brain_pin_e controlPin;Acts as EN pin in two-wire mode
+	brain_pin_e controlPin;+Acts as EN pin in two-wire mode
 	brain_pin_e disablePin;
 end_struct
 	
 	idle_hardware_s idle;
 
-	float manIdlePosition;value between 0 and 100 used in Manual mode;"%", 1, 0, 0, 100, 0
+	float manIdlePosition;+Value between 0 and 100 used in Manual mode;"%", 1, 0, 0, 100, 0
```

Comment:
not useful tooltip

Reply:
Do we have a better tip? That one was the existing deactivated tip